### PR TITLE
Addition of Connection ID parameter to main output log

### PR DESCRIPTION
**Pull request by yzninja**
I attempted to address issue #33 by adding the Connection ID value to entries in the main output log. I tested the change and the Connection ID should appear for all entries i.e. INFO, WARNING, ERROR and CRITICAL.
Please let me know if I missed anything.
PS. I wasn't sure how to assign the pull request directly to the issue.

### CODE REVIEW

File: nogotofail/mitm/connection/handlers/connection/log.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ class LoggingHandler(base.BaseConnectionHandler):
 
     FORMAT = (
         "[%(client_addr)s:%(client_port)s<=>%(server_addr)s:%(server_port)s "
-        "%(class)s](%(app_str)s) %(message)s")
+        "connection_id=\"%(connection_id)s\" %(class)s](%(app_str)s) %(message)s")
```
Comment (chadbrubaker) [inline nogotofail/mitm/connection/handlers/connection/log.py]: This looks a bit different format wise than the rest of the message, maybe strip out the connection=\"<...>\"?

**Comment by chadbrubaker** [comment]
Can you squash your commits into a single commit

**Comment by yzninja** [comment]
Sorry Chad I'm still a newbie with Github. Is there a simple way to squash all changes into a single commit from the github web ui or a cloned fork in linux?

**Comment by yzninja** [comment]
Sorry Chad. I tried squashing the commits into 1 commit - but it looks like I made a mess :( Is it best to start a new pull request?

**Comment by chadbrubaker** [comment]
You should be able to squash it and force push the branch and keep this pull request.
Try:
git checkout <branch you did your work on>
git reset --hard  2ef8cc0
git merge --squash  cfe9ebc
git push --force github <branch>

**Comment by yzninja** [comment]
Thanks Chad. I tried the commands on my local linux machine running git. I
could push the code back to my "dev" branch but had trouble squashing
multiple commits.
I'll keep trying - I'm fairly new to git.

**Comment by chadbrubaker** [comment]
If it helps you can start a new branch, pull all your logging changes into it as a single commit and upload a new pull request.

**Comment by yzninja** [comment]
Chad I ended up staying with the same pull request. I squashed alot of the commits but still have 2 commits, I'd like to merge into one. I wasn't sure how to do this in git when the aren't consecutive.

**Comment by yzninja** [comment]
I think I managed to squash the 2 commits into a single commit. Let me know if it didn't work.

**Comment by yzninja** [comment]
I chose to close this pull request. I tidied up the commit history for the branch and it would be cleaner to start again.

**Pull request closed without merging**